misk/src/main/kotlin/misk/web/jetty/MeasuredWindowRateControl.kt

```diff
@@ -0,0 +1,62 @@
+package misk.web.jetty
```

Comment:
Don’t forget to include Jetty’s copyright header for code derived from that project!

```suggestion
//
// ========================================================================
// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
//
// This program and the accompanying materials are made available under the
// terms of the Eclipse Public License v. 2.0 which is available at
// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
// which is available at https://www.apache.org/licenses/LICENSE-2.0.
//
// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
// ========================================================================
//
package misk.web.jetty
```

Reply:
I copied from this file:
https://github.com/jetty/jetty.project/blob/41d8f51edd33e81f6658e025488f13fad94d95a6/jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/WindowRateControl.java#L33